tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +33,9 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert items == []
```

Comment:
👍 In tests, I like the asserts to be as specific as we can make them. We are told this should return an empty list in this situation, and your check literally checks for an empty list on line 36. This ensures the type is correct (a list) and that it has no contents.